### PR TITLE
[Ready][Until ANOTHER FUCKIMNG MERGE CREATES CONFLICTS GOD FUCK] Reflavors a lot of Bloodsucker stuff to be more in line with it's source 

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.

## Why It's Good For The Game

My love of VTM cannot stand things such as 'blah-sucker', 'vassels' or 'archive of the kindred'.

The Blah-Sucker language has been renamed to Enochian, reflecting the First City of Vampires, Enoch, in the land of Nod east of Eden where the Dark Father Caine was exiled from.

(Front facing) mentions of vassels have been replaced with the accurate Ghoul. They drink vitae to establish a bond with their master, not blood, vampires do not produce blood, they drink vitae.

The Archive of the Kindred has been replaced with the Book of Nod. Well, a version of it. Cainites are protective of the hidden knowledge kept within and would be unhappy with the mortals (you) having a copy.

Objectives have been re-written to mention lore aspects.

## Proof Of Testing

:cl:
spellcheck: Various Bloodsucker things have been renamed after the Inconnu paid a visit.
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
@Majkl-J @Arturlang I done a very stupid thing and web resolved like six conflicts. Please god make sure I didn't accidentally break anything.

File: modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/life.dm [hunk 1]
```diff
@@ -281,9 +282,9 @@
 /// FINAL DEATH
 /datum/antagonist/bloodsucker/proc/HandleDeath()
 	if(QDELETED(owner.current))
-		if(length(vassals))
-			free_all_vassals()
-		vassals = list()
+		if(length(ghoulss))
```
Comment (theselfish) [inline modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/life.dm:285]:
```suggestion
		if(length(ghouls))
```

**Comment by theselfish** [comment]
@StrangeWeirdKitten I've been pestering Waterpig but, well I feel bad. You're the only other maintainer I know.

File: modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm [hunk 2]
```diff
@@ -522,16 +522,16 @@
...
+	var/datum/objective/bloodsucker/haven/haven_objective = new
+	haven_objective.owner = owner
+	objectives += haven_objective
 	// Survive Objective
 	var/datum/objective/survive/bloodsucker/survive_objective = new
 	survive_objective.owner = owner
 	objectives += survive_objective
 
-	// Objective 1: Vassalize a Head/Command, or a specific target
+	// Objective 1: Ghoulize a Head/Command, or a specific target
```
Comment (Arturlang) [inline modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm:534]: Ghoul, not ghoulize

File: modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/conversion.dm [hunk 3]
```diff
@@ -1,83 +1,83 @@
...
  * Checks if the target has antag datums and, if so,
- * are they allowed to be Vassalized, or not, or banned.
+ * are they allowed to be Ghoulized, or not, or banned.
  * Args:
  * target - The person we check for antag datums.
  */
 /datum/antagonist/bloodsucker/proc/AmValidAntag(mob/target)
 	if(HAS_TRAIT(target, TRAIT_UNCONVERTABLE))
-		return VASSALIZATION_BANNED
+		return GHOULIZATION_BANNED
```
Comment (Arturlang) [inline modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/conversion.dm:9]: Maybe it'd be better if we just called ghoulization ghouling, instead?

File: code/modules/admin/sql_ban_system.dm [hunk 4]
```diff
@@ -435,7 +435,7 @@
 				ROLE_BORER, // SKYRAT EDIT ADDITION
 				ROLE_ASSAULT_OPERATIVE, // SKYRAT EDIT ADDITION
 				ROLE_BLOODSUCKER,// BUBBER EDIT
-				ROLE_VASSAL, // BUBBER EDIT
+				ROLE_GHOUL, // BUBBER EDIT
```
Comment (Arturlang) [inline code/modules/admin/sql_ban_system.dm:438]: This might break vassalbans

Comment (Arturlang) [reply code/modules/admin/sql_ban_system.dm:438]: Unless someone updates the database

File: modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm (hunk 2 above)
Comment (theselfish) [reply modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm:534]: Ghoulification sounds better. I'll go w/ that.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
Keep the bans the same define.

**Comment by theselfish** [comment]
> Keep the bans the same define.

Did I not change that back?

**Comment by theselfish** [comment]
> Keep the bans the same define.

Yeah the role bans got changed back as far as I can see.

**Pull request merged**